apps/backoffice-v2/src/pages/Entity/components/Case/case-action-variants/WebsiteMonitoringCaseActions/WebsiteMonitoringCaseActions.tsx

```diff
@@ -0,0 +1,156 @@
+import { Badge } from '@ballerine/ui';
+import { FunctionComponent } from 'react';
+
+import { Input } from '@/common/components/atoms/Input/Input';
+import { Label } from '@/common/components/atoms/Label/Label';
+import { Select } from '@/common/components/atoms/Select/Select';
+import { SelectContent } from '@/common/components/atoms/Select/Select.Content';
+import { SelectItem } from '@/common/components/atoms/Select/Select.Item';
+import { SelectTrigger } from '@/common/components/atoms/Select/Select.Trigger';
+import { SelectValue } from '@/common/components/atoms/Select/Select.Value';
+import { websiteMonitoringTagsToData } from '@/pages/Entity/components/Case/case-action-variants/WebsiteMonitoringCaseActions/constants';
+import { useWebsiteMonitoringCaseActions } from '@/pages/Entity/components/Case/case-action-variants/WebsiteMonitoringCaseActions/hooks/useWebsiteMonitoringCaseActions/useWebsiteMonitoringCaseActions';
+import { DialogClose } from '@radix-ui/react-dialog';
+import { AssignDropdown } from '../../../../../../common/components/atoms/AssignDropdown/AssignDropdown';
+import { Button } from '../../../../../../common/components/atoms/Button/Button';
+import { Dialog } from '../../../../../../common/components/organisms/Dialog/Dialog';
+import { DialogContent } from '../../../../../../common/components/organisms/Dialog/Dialog.Content';
+import { DialogDescription } from '../../../../../../common/components/organisms/Dialog/Dialog.Description';
+import { DialogFooter } from '../../../../../../common/components/organisms/Dialog/Dialog.Footer';
+import { DialogHeader } from '../../../../../../common/components/organisms/Dialog/Dialog.Header';
+import { DialogTitle } from '../../../../../../common/components/organisms/Dialog/Dialog.Title';
+import { DialogTrigger } from '../../../../../../common/components/organisms/Dialog/Dialog.Trigger';
+import { ctw } from '../../../../../../common/utils/ctw/ctw';
+import { tagToBadgeData } from '../../consts';
+import { IActionsProps } from '../../interfaces';
+
+/**
+ * @description To be used by {@link Case}. Displays the entity's full name, avatar, and handles the reject/approve mutation.
+ *
+ * @param props
+ * @param props.id - The id of the entity, passed into the reject/approve mutation.
+ * @param props.fullName - The full name of the entity.
+ * @param props.showResolutionButtons - Whether to show the reject/approve buttons.
+ *
+ * @see {@link Case}
+ * @see {@link Avatar}
+ *
+ * @constructor
+ */
+export const WebsiteMonitoringCaseActions: FunctionComponent<IActionsProps> = ({
+  id,
+  fullName,
+  showResolutionButtons,
+}) => {
+  const {
+    tag,
+    assignedUser,
+    authenticatedUser,
+    isLoadingCase,
+    assignees,
+    formValues,
+    isLoading,
+    isActionsDisabled,
+    handleResolutionActionChange,
+    handleResolutionCommentChange,
+    handleSubmit,
+    onMutateAssignWorkflow,
+  } = useWebsiteMonitoringCaseActions({ workflowId: id, fullName });
```

Comment:
Please follow the convention for a component's business logic hook. 🙏🏼 
```suggestion
  } = useWebsiteMonitoringCaseActionsLogic({ workflowId: id, fullName });
```

Reply:
P1